src/lib/APIs.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL: string = "https://upskilling-egypt.com:3003/api/v1";
```

Comment:
We can also improve on this by using axios.create and passing the url, and it will be used in any place we use the axiosInstance.